src/components/StoreVersionDialog.test.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Alert } from "react-native";
+import VersionCheck from "react-native-version-check";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import StoreVersionDialog from "./StoreVersionDialog";
+import * as storage from "../integrations/storage";
+import * as date from "../lib/date";
+
+const nextTick = () => new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
Do you think we can use [`jest.runAllTicks()`](https://facebook.github.io/jest/docs/en/jest-object.html#jestrunallticks) instead?

Reply:
I didn't know about this, but I tried it out and it gave me a weird warning and didn't work.

I've simplified it a little to use `process.nextTick` though, since I only just found out about this too.